mkdocs.yml

```diff
@@ -146,16 +146,20 @@ nav:
           - 'Features': unified/novacustom/features.md
           - 'Hardware Compatibility List': unified/novacustom/hcl.md
           - 'Firmware transition': unified/novacustom/firmware-transition.md
-          - 'V54 14th Gen':
+          - 'V540TU 14th Gen':
               - 'Releases': variants/novacustom_v540tu/releases.md
               - 'Hardware Configuration Matrix': variants/novacustom_v540tu/hardware-matrix.md
               - 'Test matrix': variants/novacustom_v540tu/test-matrix.md
               - 'Openness score': variants/novacustom_v540tu/openness-score.md
-          - 'V56 14th Gen':
+          - 'V540ND 14th Gen':
```

Comment:
```suggestion
          - 'V540TND 14th Gen':
```

Reply:
addressed here: [ce52402](https://github.com/Dasharo/docs/pull/860/commits/ce52402c0a72c81d06e346c2c0cee7a18ecbef2c)

Reply:
Specific models, like `V540TND` still exist in the `recovery` document. I wanted to join them but there are seperate photos for `TND` and `TNE` models so I decided to skip that for now.